src/pages/Homepage.module.css

```diff
@@ -1,4 +1,5 @@
 .homepage {
   background-image: url("../assests/images/homepage-background.jpeg");
   background-size: cover;
+  min-height: 62vh;
```

Comment:
Same as the other related comments - what is 62? Will this look good on various screen sizes?

Reply:
for this, the purpose was to make sure the footer didn't get too close to the header. But yes, it should look good